cmd/wego/main.go

```diff
@@ -37,7 +78,6 @@ func configureLogger() {
 func main() {
 	fluxBin.SetupFluxBin()
 	rootCmd.PersistentFlags().BoolVarP(&options.verbose, "verbose", "v", false, "Enable verbose output")
-	rootCmd.PersistentFlags().String("namespace", "wego-system", "gitops runtime namespace")
```

Comment:
We need to keep the flag in the command, even if we remove it from the examples

Reply:
This has been fixed.